test/Microsoft.SqlTools.ServiceLayer.UnitTests/SchemaCompare/SchemaCompareTests.cs

```diff
@@ -0,0 +1,89 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System;
+using Microsoft.SqlTools.ServiceLayer.SchemaCompare;
+using Xunit;
+
+namespace Microsoft.SqlTools.ServiceLayer.UnitTests.SchemaCompare
+{
+    public class SchemaCompareTests
+    {
+        [Fact]
+        public void FormatScriptAddsGo()
+        {
+            string script = "EXECUTE sp_addextendedproperty @name = N'MS_Description', @value = N'Primary key for AWBuildVersion records.', @level0type = N'SCHEMA', @level0name = N'dbo', @level1type = N'TABLE', @level1name = N'AWBuildVersion', @level2type = N'COLUMN', @level2name = N'SystemInformationID';";
+            Assert.DoesNotContain("GO", script);
+            string result = SchemaCompareOperation.FormatScript(script);
+            Assert.Contains("GO", result);
```

Comment:
nit: But this is redundant with the "EndsWith"

Reply:
removed